test/scale-infra/INSTRUCTIONS.md

```diff
@@ -0,0 +1,229 @@
+    kubectl --kubeconfig=$PWD/.data/admin.conf --context wds1 get ns
+    ```
+
+    Output:
+    ```console
+    NAME                 STATUS   AGE
+    kube-system          Active   29m
+    kube-public          Active   29m
+    kube-node-lease      Active   29m
+    default              Active   29m
+    kubestellar-report   Active   28m
+    ```
+
+4. Create the WEC hosting instances:
+
+    Set the following variables:
+    ```console
+    CLUSTER_NAME="wec"
+    NUM_HOSTING_INSTANCES=2
+    EC2_INSTANCE_TYPE="t2.xlarge"
+    ```
+
+    Then, deploy the WEC infra:
+    ```bash
+    ansible-playbook create-ec2.yaml -e "cluster_name=$CLUSTER_NAME region=$REGION vpc_name=$VPC aws_key_name=$EC2_SSH_PUBLIC_KEY wecs_hosting_instances=$NUM_HOSTING_INSTANCES instance_type=$EC2_INSTANCE_TYPE archt=$ARCH ec2_image=$EC2_AMI" 
+    ```
+
+    Use the variable `NUM_HOSTING_INSTANCES` to specify the number of ec2 instances to be created to host WEC kind clusters.
+    
+    Upon completion of the script's execution, an Ansible inventory file containing the IP addresses of the ec2 WEC hosting instances will be generated in the present directory at `.data/hosts_wec`.
+
+5. Create the WEC kind clusters:
+
+    a) Update the inventory of the WEC instances to include the IP address of a master node from the K8s cluster created in step 1 above:
+
+    First, find the required master's IP address info in the following file: `.data/hosts_core`.
+
+    Next, use the following command to see the content of the WEC ansible inventory:
+    ```bash
+    cat .data/hosts_wec
+    ```
+
+    Sample output: 
+    ```console
+    [masters]
+
+    [add_workers]
+    worker1 ansible_host=192.168.56.1
+    worker1 ansible_host=192.168.56.2
+    ```
+
+    Lastly, edit the contents of the file `.data/hosts_wec` to include the IP address and name of a master node in the [masters] section, for example:
+    ```console
+    [masters]
+    master1 ansible_host=192.168.56.20
+
+    [add_workers]
+    worker1 ansible_host=192.168.56.1
+    worker1 ansible_host=192.168.56.2
+    ```
+
+    b) Create WEC kind clusters and connect to KS Core cluster
+
+    ```
+    ansible-playbook -i .data/hosts_wec deploy_ks_wec.yaml --ssh-common-args="-o StrictHostKeyChecking=no" -e "region=$REGION vpc_name=$VPC num_wecs=1"
+    ```
+
+    Use the input paramater `num_wecs` to specify the number of kind clusters to be created for each WEC Hosting Instances. The above command creates kind WEC clusters and connects them to the KubeStellar core cluster created in step 1. Furthermore, it attaches a [KWOK](https://github.com/kubernetes-sigs/kwok) fake node to each kind cluster.
+
+
+    c) Access the WEC kind clusters from Ansible control machine:
+
+    A kubeconfig file is generated for each WEC hosting instances at the following directory `.data/wecs_config/`. For example:
+
+    ```bash
+    ls .data/wecs_config/
+    ``` 
+
+    Output:
+    ```console
+    192-168-56-1.conf	192-168-56-2.conf
```

Comment:
Testing commit b00fdae41b4883c9af6c171759c855153fb6fe08, I get global IP addresses here, not RFC 1918 ones.

Reply:
Yes! that's the expected result. I did not want to expose valid AWS public ip addresses on my README instructions, so I used those one just as an example.

Reply:
There are other ways to make example IP addresses that do not suggest the wrong class and are obviously not real. For example, 13.300.1.3. For better examples, see https://datatracker.ietf.org/doc/html/rfc5737